src/UI/Component/Layout/Page/Factory.php

```diff
@@ -44,6 +44,14 @@ interface Factory
      *     7: The Standard Page's short title SHOULD reference the current ILIAS installation.
      *     8: The Standard Page's view title SHOULD give a good hint to the current view.
      *
+     * rules:
+     *   accessibility:
+     *     1: >
+     *        Scrollable areas of the Standard Page MUST be scrollable by only using
+     *        the keyboard.
+     *     2: >
+     *        The element carrying the focus (e.g. by tabbing) in the Standard Layout
+     *        MUST always be visibly marked (e.g. by some sort of highlighting).
```

Comment:
In the Standard Layout the active element carrying the focus. Focus can be moved by  tab, shift+tab keys and up and down arrows.  
The focus MUST move in a logical and predictable order. 
The active element carrying the focus MUST have a visual focus indicator.